src/tools/storage-perf/StoragePerfTool.cpp

```diff
@@ -177,7 +186,7 @@ class Perf {
 
     void getVerticesTask() {
         auto* evb = threadPool_->getEventBase();
-        auto f = client_->getVertexProps(defaultSpaceId_, randomVertices(), randomCols())
+        auto f = client_->getVertexProps(FLAGS_default_space_id, randomVertices(), randomCols())
```

Comment:
is random() fast enough?
any benchmark?

Reply:
Looks good now.